nucliadb/tests/ingest/fixtures.py

```diff
@@ -185,14 +185,12 @@ def learning_config():
 
 @pytest.fixture(scope="function")
 async def knowledgebox_ingest(storage, maindb_driver: Driver, shard_manager, learning_config):
-    kbid = str(uuid.uuid4())
-    kbslug = str(uuid.uuid4())
-    async with maindb_driver.transaction() as txn:
-        model = SemanticModelMetadata(
-            similarity_function=upb.VectorSimilarity.COSINE, vector_dimension=len(V1)
-        )
-        await KnowledgeBox.create(txn, kbslug, model, uuid=kbid)
-        await txn.commit()
+    kbid = KnowledgeBox.new_unique_kbid()
+    kbslug = "slug-" + str(uuid.uuid4())
+    model = SemanticModelMetadata(
+        similarity_function=upb.VectorSimilarity.COSINE, vector_dimension=len(V1)
+    )
+    await KnowledgeBox.create(maindb_driver, kbid=kbid, slug=kbslug, semantic_model=model)
```

Comment:
Are you still using the deprecated creation on purpose, so we still test that code path?

Reply:
Yes, but I can use the new one if you prefer